libs/blocks/merch-card/merch-card.js

```diff
@@ -70,17 +77,21 @@ const parseContent = (el, merchCard) => {
   innerElements.forEach((element) => {
     let { tagName } = element;
     if (isHeadingTag(tagName)) {
-      let slotName = textStyles[tagName];
+      let slotName = TEXT_STYLES[tagName];
       if (slotName) {
         if (['H2', 'H3', 'H4', 'H5'].includes(tagName)) {
-          if (tagName === 'H2') {
-            headingMCount += 1;
-          }
-          if (headingMCount === 2 && merchCard.variant === MINI_COMPARE_CHART) {
-            slotName = 'heading-m-price';
+          if (HEADING_MAP[merchCard.variant] && HEADING_MAP[merchCard.variant][tagName]) {
```

Comment:
Nit: could optional chaining be used? `HEADING_MAP[merchCard.variant]?.[tagName]`

Reply:
fun i remember you already making me that (good) suggestion in the past on createTag api...people don't learn

Reply:
I just have one comment up my sleeve and reuse it 😛